CONTRIBUTING.md

```diff
@@ -3,24 +3,24 @@
 ## Create the docs locally
 
 ``` python
-conda env update -f environment.yml
+conda env create --file environment.yml
```

Comment:
Either way works on my end as well, but the `create` option is clearer about what the script is doing, since `update` implies it is modifying something that is already existing which can be confusing is someone is less familiar with conda

Reply:
Yeah, that's fair.  Sometimes the messages when you already have the environment are a bit confusing as well, but I think this is fine.